Documentation/flex-to-csi-migration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Flex Migration
+weight: 11900
+indent: true
+---
+
+# Flex to CSI Migration
+
+In Rook v1.8, the Flex driver has been deprecated. Before updating to v1.8, any Flex volumes created in previous versions of Rook will need to be converted to Ceph-CSI volumes.
+
+The tool [persistent-volume-migrator](https://github.com/ceph/persistent-volume-migrator) will help automate migration of Flex rbd volumes to Ceph-CSI volumes.
+
+> **Note** Migration of CephFS FlexVolumes is not supported for now.
+
+## Migration Preparation
+
+1. Rook v1.7.x is required. If you have a previous version of Rook running, follow the [upgrade guide](https://rook.io/docs/rook/v1.7/ceph-upgrade.html) to upgrade from previous releases until on v1.7.x.
```

Comment:
Let's go ahead and mention the latest release, then we can update the recommendation with each 1.7.x release as well
```suggestion
1. Rook v1.7.8 is required. If you have a previous version of Rook running, follow the [upgrade guide](https://rook.io/docs/rook/v1.7/ceph-upgrade.html) to upgrade from previous releases until on v1.7.8.
```

Reply:
changed